src/code/z_demo.c

```diff
@@ -1,99 +1,1634 @@
+            }
+            break;
+        case 0x7:
+            if (isStartFrame) {
+                loadedScene = globalCtx->loadedScene;
+                if (loadedScene->titleTextId != 0) {
+                    func_80151A68(globalCtx, loadedScene->titleTextId);
+                }
+            }
+            break;
+        case 0x8:
+            func_8019F128(NA_SE_EV_EARTHQUAKE_LAST - SFX_FLAG);
+            if (isStartFrame) {
+                sCutsceneQuakeIndex = Quake_Add(GET_ACTIVE_CAM(globalCtx), 6);
+                Quake_SetSpeed(sCutsceneQuakeIndex, 22000);
+                Quake_SetQuakeValues(sCutsceneQuakeIndex, 6, 4, 0, 0);
+                Quake_SetCountdown(sCutsceneQuakeIndex, 800);
+            }
+            break;
+        case 0x9:
+            if (isStartFrame) {
+                Quake_Init();
+            }
+            break;
+        case 0xA:
+            D_801F6D30.r = 255;
+            D_801F6D30.g = 255;
+            D_801F6D30.b = 255;
+            D_801F6D30.a = 255 * progress;
+            break;
+        case 0xB:
+            D_801F6D30.r = 255;
+            D_801F6D30.g = 180;
+            D_801F6D30.b = 100;
+            D_801F6D30.a = 255 * progress;
+            break;
+        case 0xC:
+            globalCtx->roomCtx.currRoom.segment = NULL;
+            break;
+        case 0xD:
+            if (globalCtx->state.frames & 8) {
+                if (globalCtx->envCtx.lightAdjustments.ambientColor[0] < 40) {
+                    globalCtx->envCtx.lightAdjustments.ambientColor[0] += 2;
+                    globalCtx->envCtx.lightAdjustments.diffuseColor1[1] -= 3;
+                    globalCtx->envCtx.lightAdjustments.diffuseColor1[2] -= 3;
+                }
+            } else {
+                if (globalCtx->envCtx.lightAdjustments.ambientColor[0] > 2) {
+                    globalCtx->envCtx.lightAdjustments.ambientColor[0] -= 2;
+                    globalCtx->envCtx.lightAdjustments.diffuseColor1[1] += 3;
+                    globalCtx->envCtx.lightAdjustments.diffuseColor1[2] += 3;
+                }
+            }
+            break;
+        case 0xE:
+            globalCtx->unk_18845 = 1;
+            break;
+        case 0xF:
+            globalCtx->unk_18845 = 0;
+            break;
+        case 0x10:
+            if (isStartFrame) {
+                globalCtx->envCtx.sandstormState = 1;
+            }
+            func_8019F128(NA_SE_EV_SAND_STORM - SFX_FLAG);
+            break;
+        case 0x11:
+            gSaveContext.unk_3F58 = 1;
+            break;
+        case 0x12:
+            if (!gSaveContext.isNight) {
+                time = gSaveContext.time;
+                gSaveContext.time = time - (u16)REG(15);
+            } else {
+                time = gSaveContext.time;
+                gSaveContext.time = time - (u16)(2 * REG(15));
+            }
+            break;
+        case 0x13:
+            // AudioOcarina_PlayLongScarecrowAfterCredits
```

Comment:
That's how @engineer124 named that function on his audio branches.
I didn't rename it to avoid conflicts with his stuff. But I can rename it if he is ok with it